superset/sqllab/api.py

```diff
@@ -236,6 +240,7 @@ def format_sql(self) -> FlaskResponse:
             return self.response_400(message=error.messages)
 
     @expose("/export/<string:client_id>/")
```

Comment:
@jessie-ross Thanks for the great feature proposal. I was reading through the SIP, and thought I might suggest another option here which would be to use the existing api "/export/<string:client_id>/" and pass the export type with a query string `?format=csv` for example. That could potentially simplify this. It seems like the request/response schemas should be fairly similar, with just a switch in the command that is run.

Reply:
I'm happy to go either way on the url format, the two are equivalent:
```
/export/<client-id>/?format=csv
/export/<client-id>/?format=google-sheets
/export/<client-id>/   (csv, deprecated)
```

```
/export/<client-id>/csv/
/export/<client-id>/google-sheets/
/export/<client-id>/   (csv, deprecated)
```

Reply:
Great. Maybe another option is to make csv the default format if none is passed. Then we wouldn't need to deprecate the existing api.

Reply:
I'm happy to do that too, it is really an overall API design point, and I'm not sure how Superset does things.

In my mind deprecated doesn't mean it has to ever be removed, so it never has to be a breaking change.

Reply:
Since csv is default if none is passed (meaning no breaking changes,) and the SIP includes the current format, I'm going to go with that.